sycl/test-e2e/Graph/README.md

```diff
@@ -0,0 +1,35 @@
+## Overview
+
+The "Graph" directory contains tests for the
+[sycl_ext_oneapi_graph](../../doc/extensions/experimental/sycl_ext_oneapi_graph.asciidoc)
+extension.
+
+Each subdirectory contains a `lit.local.cfg` file. This file sets the `lit`
+option `config.required_features` to the graph aspect required to run the tests:
```

Comment:
The `UnsupportedDevice` and `Inputs` directories don't have a `lit.local.cfg` file, so perhaps this should be "Most" or "Many" (or something to that effect) instead of "Each" subdirectory.

Reply:
Change it to "Many"